### ISSUE TITLE
Internal Error - In Mixins.php line 71:

### ISSUE THREAD

**Issue by Nizari** [nunomaduro/larastan#133]
- Larastan Version: v0.2.9
- `--level` used: 0,1,2

### Description:
```
» php artisan code:analyse -l 0
PHPStan crashed in the previous run probably because of excessive memory consumption.
It consumed around 10 MB of memory.


To avoid this issue, allow to use more memory with the --memory-limit option.
 43/43 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------ --------------------------------------------------------------
  Line   app/Http/Controllers/AbstractCrudController.php
 ------ --------------------------------------------------------------
         Internal error: Class https does not exist
         Run PHPStan with --debug option and post the stack trace to:
         https://github.com/phpstan/phpstan/issues/new
 ------ --------------------------------------------------------------


 [ERROR] Found 1 error


------------------------------------------------------------
» php artisan code:analyse -l 2 --debug                                                                             
~/atabase3/app/Console/BaseGeneratorCommand.php
~/atabase3/app/Console/Commands/ControllerMakeCommand.php
~/atabase3/app/Console/Commands/CreateUser.php
~/atabase3/app/Console/Commands/ModelMakeCommand.php
~/atabase3/app/Console/Commands/PolicyMakeCommand.php
~/atabase3/app/Console/Commands/RequestMakeCommand.php
~/atabase3/app/Console/Commands/ResourceMakeCommand.php
~/atabase3/app/Console/Commands/SeederMakeCommand.php
~/atabase3/app/Console/Commands/TestMakeCommand.php
~/atabase3/app/Console/Commands/TransformerMakeCommand.php
~/atabase3/app/Console/Kernel.php
~/atabase3/app/Exceptions/Handler.php
~/atabase3/app/Exceptions/Repositories/CreateErrorException.php
~/atabase3/app/Exceptions/Repositories/UpdateErrorException.php
~/atabase3/app/helpers.php
~/atabase3/app/Http/Controllers/AbstractCrudController.php

In Mixins.php line 71:

  Class https does not exist
```

**Comment by Nizari** [comment]
Apparently it breaks here
```
    /**
     * @param Model $model
     *
     * @return BaseRepository
     */
    protected function initRepositoryClass(Model $model = null): BaseRepository
    {
        if (\is_null($model)) {
            $modelClass = $this->modelClass();
            $model = new $modelClass();
        }
        $repositoryClass = $this->repositoryClass();

        return new $repositoryClass($model);
    }
    ```

**Comment by hartman** [comment]
I have something similar which i described here https://github.com/nunomaduro/larastan/issues/26#issuecomment-418148474

It's caused by @mixin or @see annotations added in the phpdoc of your classes, which the reflection cannot resolve.

See code starting: https://github.com/nunomaduro/larastan/blob/master/src/Methods/Pipes/Mixins.php#L60
failing at:
https://github.com/nunomaduro/larastan/blob/master/src/Methods/Pipes/Mixins.php#L71

**Comment by nunomaduro** [comment]
@hartman What do you mean? Can you share the code that fails?

**Comment by hartman** [comment]
@nunomaduro OK, this is an example model that has this problem. I've stripped it of some details:
The class is annotated by `php artisan ide-helper:models` and it breaks on the `@mixin \Eloquent`

Installed as specified in 
`https://github.com/barryvdh/laravel-ide-helper#lumen-install`

It is possibly related to: https://github.com/barryvdh/laravel-ide-helper/issues/611


```
<?php namespace App\Models;

use Illuminate\Database\Eloquent\Model;

/**
 * App\Models\Account
 *
 * @property int $id
 * @property string|null $name
 * @property string|null $external_id
 * @property \Carbon\Carbon|null $created_at
 * @property \Carbon\Carbon|null $updated_at
 * @property-read \Illuminate\Database\Eloquent\Collection|\App\Models\Client[] $clients
 * @method static \Illuminate\Database\Eloquent\Builder|\App\Models\Account whereCreatedAt($value)
 * @method static \Illuminate\Database\Eloquent\Builder|\App\Models\Account whereExternalId($value)
 * @method static \Illuminate\Database\Eloquent\Builder|\App\Models\Account whereId($value)
 * @method static \Illuminate\Database\Eloquent\Builder|\App\Models\Account whereName($value)
 * @method static \Illuminate\Database\Eloquent\Builder|\App\Models\Account whereUpdatedAt($value)
 * @mixin \Eloquent
 */
class Account extends Model
{
    /**
     * The table associated with the model.
     *
     * @var string
     */
    protected $table = 'accounts';

    /**
     * @return \Illuminate\Database\Eloquent\Relations\HasMany
     */
    public function clients()
    {
        return $this->hasMany(Client::class, 'account_id');
    }
}

```

I tinkered about with Mixins.php:71 and when doing something try catching the exception like:
```
$mixins = array_filter(
            $mixins,
            function ($mixin) use ($classReflection) {
                try {
                    return (new \ReflectionClass($mixin))->getName() !== $classReflection->getName();
                } catch (\ReflectionException $e) {
                    error_log('unable to load ' .$mixin);
                }
            }
        );
```
Then larastan completes as normal. (well, with errror_logs for all my models, but whatever).

**Comment by nunomaduro** [comment]
Please perform a Pull Request, returning false on catch cases. 🙂

**Comment by nunomaduro** [comment]
Thanks issue has being fixed on the version `v0.2.10`. Thanks to @hartman!
